solr/solrj-streaming/src/java/org/apache/solr/client/solrj/io/stream/SelectStream.java

```diff
@@ -308,6 +327,13 @@ public Tuple read() throws IOException {
       workingForEvaluators.put(fieldName, original.get(fieldName));
       if (selectedFields.containsKey(fieldName)) {
         workingToReturn.put(selectedFields.get(fieldName), original.get(fieldName));
+      } else {
+        for (String globPattern : selectedFieldGlobPatterns) {
+          if (GlobPatternUtil.matches(globPattern, fieldName)) {
```

Comment:
Shouldn't SelectStream also use SolrReturnFields and not use lower level GlobPattern stuff (it's something SRF can handle)?

Disclaimer: I haven't looked at this PR in a long time.

Reply:
`solrj-streaming` does not currently have a dependency on `core`, in fact currently I think `core` depends on `solrj-streaming`. I didn't want to refactor SolrReturnFields to live elsewhere given the scope of this PR so not using that.

Reply:
Oh right; of course.